doc/ref/configuration/master.rst

```diff
@@ -3121,6 +3121,23 @@ order will be used.
 
     syndic_failover: random
 
+. conf_master:: syndic_forward_all_events
+
+``syndic_forward_all_events`
+-------------------
+
+.. versionadded:: Nitrogen
+
+Default: ``False``
+
+Option on multi-syndic or single when connected to multiple masters to be able to
+sent events to all connected masters.
```

Comment:
Can you fix up this small typo, too? This should be ``send`` instead of ``sent``.